web/src/settings_org.js

```diff
@@ -874,6 +929,17 @@ export function populate_data_for_request(
                     };
                     continue;
                 }
+
+                if (property_name === "allow_resolve_topic") {
+                    let user_group;
+                    if (input_value === false) {
+                        user_group = user_groups.get_user_group_from_name("role:nobody");
+                        data.resolve_topic_group = user_group.id;
+                    } else {
```

Comment:
When submitting request, if `allow_resolve_topic` checkbox is checked, we send a request of `resolve_topic_group = dropdown value`. Else, we send request of `resolve_topic_group` = `role:nobody`.id.